Interesting error:
```
Warning, treated as error:
autodoc: failed to import module 'tft_gizmo' from module 'adafruit_gizmo'; the following exception was raised:
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/sphinx/ext/autodoc/importer.py", line 32, in import_module
    return importlib.import_module(modname)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/travis/build/adafruit/Adafruit_CircuitPython_Gizmo/adafruit_gizmo/tft_gizmo.py", line 40, in <module>
    class TFT_Gizmo(ST7789):
  File "/home/travis/build/adafruit/Adafruit_CircuitPython_Gizmo/adafruit_gizmo/tft_gizmo.py", line 43, in TFT_Gizmo
    def __init__(self, *, spi=None, cs=board.RX, dc=board.TX,
AttributeError: module 'board' has no attribute 'RX'
The command "cd docs && sphinx-build -E -W -b html . _build/html && cd .." exited with 2.
```

I wonder if Blinka might need this added just so it can test this.

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -20,7 +20,7 @@ extensions = [
 # Uncomment the below if you use native CircuitPython modules such as
 # digitalio, micropython and busio. List the modules you use. Without it, the
 # autodoc module docs will fail to generate with a warning.
-autodoc_mock_imports = ["adafruit_st7789", "digitalio", "busio", "displayio"]
+autodoc_mock_imports = ["adafruit_st7789", "digitalio", "busio", "displayio", "board"]
 
 
 intersphinx_mapping = {'python': ('https://docs.python.org/3.4', None),'BusDevice': ('https://circuitpython.readthedocs.io/projects/busdevice/en/latest/', None),'CircuitPython': ('https://circuitpython.readthedocs.io/en/latest/', None)}

